spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -885,10 +894,9 @@ Obtaining the previous 10 products:
 
 [source,java]
 ----
-if (moreExpensive.hasContent())
-    pageRequest = pageRequest.beforeCursor(moreExpensive.getCursor(0));
-else
-    pageRequest = pageRequest.beforeKey(priceMidpoint, 1L);
+pageRequest = moreExpensive.hasContent()
+        ? pageRequest.beforeCursor(moreExpensive.getCursor(0));
+        : pageRequest = pageRequest.beforeKey(priceMidpoint, 1L);
```

Comment:
One fix to a compile issue (misplaced `;`) and one removal of an unnecessary assignment that makes it harder to read,
```suggestion
pageRequest = moreExpensive.hasContent()
        ? pageRequest.beforeCursor(moreExpensive.getCursor(0))
        : pageRequest.beforeKey(priceMidpoint, 1L);
```